src/app/research-results/[runId]/page.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, Table, TableTbody, TableTd, TableTh, TableThead, TableTr } from '@mantine/core'
+import { approveResearch, loadMetadata } from '../../utils'
+import Link from 'next/link'
+
+interface PageParams {
+    params: { runId: string }
+}
+
+export default async function ResearchResultList({ params: { runId } }: PageParams) {
+    if (runId?.startsWith('approve-')) {
+        await approveResearch(runId.substring(8))
+    }
+
+    let result = await loadMetadata()
+
+    const rows = result.map((file: any) => (
+        <TableTr key={file.runId}>
+            <TableTd>{file.time}</TableTd>
+            <TableTd>{file.runId}</TableTd>
+            <TableTd>{file.status || 'Pending'}</TableTd>
```

Comment:
It seems like this list should always only list pending results, no? Because they're deleted afterwards